### DISCUSSION TITLE
How can I use the noscript tag?

### DISCUSSION THREAD

**Discussion by miker059** [nuxt/nuxt#17680] [Questions]
Hi, friends! Please tell me how can I use the noscript tag? Previously I could have put it in app.html or in the head section in nuxt.config.js, but now it doesn't work.

**Comment by vis97c** [comment]
I think this is an important feature for any js based platform. I'll try to figure it out how to add it and update you on the task.

**Comment by vis97c** [comment]
@miker059 just added https://github.com/nuxt/framework/issues/4646 as part of the contribution flow.
